scripts/utils.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Importing this library shouldn't have side effects
+
+pkgdir=${GOPATH}/src/github.com/kubernetes-csi/csi-proxy
+
+# current_account is the current user
+# in CI, the value is `prow`
+current_account=$USER
+# windows_node is the id of the GCE Windows instance
+windows_node=$(kubectl get nodes -l kubernetes.io/os=windows -o jsonpath='{.items[*].metadata.name}')
+
+# set the default zone for the gcloud sdk
+if ! [ -z "${GCP_ZONE:-}" ]; then
+  export CLOUDSDK_COMPUTE_ZONE="$GCP_ZONE"
+fi
+
+sync_file_to_vm() {
+  gcloud compute scp $@ $windows_node:"C:\\Users\\${current_account}"
+}
+
+compile_csi_proxy() {
+  echo "Compiling CSI Proxy"
+  make -C $pkgdir build
+}
+
+compile_csi_proxy_integration_tests() {
+  echo "Compiling CSI Proxy integration tests"
+  GOOS=windows GOARCH=amd64 go test -c $pkgdir/integrationtests -o $pkgdir/bin/integrationtests.test.exe
```

Comment:
integrationtests.test.exe to integrationtests.exe?

Reply:
this is similar to the e2e test binary, from `e2e.go` we generate the binary `e2e.test`, if I don't use the output flag it'll generate it with that name `integrationtests.test.exe`, I wrote `-o ` to send it to the `bin/` folder